src/d3a/models/strategy/update_frequency.py

```diff
@@ -225,11 +227,13 @@ def reset_price_on_market_cycle(self, market):
                 continue
             try:
                 iterated_market.delete_offer(offer.id)
-
+                updated_price = offer.energy * \
+                    self.calculate_initial_sell_rate(iterated_market.time_slot)
```

Comment:
How about rounding here as well? Like in `_decrease_offer_price`?

Reply:
You are right, this should be rounded too, fixing it.